factscore/factscorer.py

```diff
@@ -108,36 +111,39 @@ def get_score(self,
             if verbose:
                 topics = tqdm(topics)
 
-            new_topics, new_generations, new_atomic_facts = [], [], []
+            atomic_facts = []
             for topic, gen in zip(topics, generations):
                 curr_afs, _ = self.af_generator.run(gen)
                 curr_afs = [fact for _, facts in curr_afs for fact in facts]
                 if len(curr_afs)==0:
-                    # abstain from answering
+                    atomic_facts.append(None)
                     continue
```

Comment:
Very minor: but maybe an `else` instead of `continue` is easier to read here.. I misunderstood the code in my first two reads